rust-toolchain

```diff
@@ -1 +1 @@
-1.75.0
+1.74.0
```

Comment:
Is there any reason to 'downgrade' the version used?

Reply:
hmm not sure I follow, in the latest `main` branch we have the 1.75 and seems to work fine, no?
(ie. https://github.com/privacy-scaling-explorations/folding-schemes/blob/main/rust-toolchain )

Reply:
Right, agree that this makes sense. Regarding the 1.75 not existing, I don't understand why then in the github ci it has been working for the past main PRs/commits.
Following this approach that you mention for the MSRV, which version should we stick to in this `rust-toolchain` file? (happy with any one, just I don't have any opinion on it but want to understand the reasoning)

Reply:
Cool, agree, keeping up to date is what I was assuming that we were doing until now (that's why I asked about the motivation of downgrading to 1.74)